### ISSUE TITLE
 Border line, seprator are not coming properly on Chrome browser

### ISSUE THREAD

**Issue by samit2015** [mozilla/pdf.js#6857]
On Chrome browser, pdf is missing lines, dividers, separators etc...any idea what needs to be done?

Image has been attached for reference. Cant add pdf as its client data. 

Any help wld be great for me to fix this?
![error](https://cloud.githubusercontent.com/assets/16206635/12262018/f96ecf4e-b8f2-11e5-8efd-029877b4db75.jpg)

**Comment by samit2015** [comment]
We can see missing lines only after keep zooming....I need to fix this by code for 100%

**Comment by timvandermeij** [comment]
Since there is no PDF file for us to test with, I'm closing this issue as a duplicate of #2722 based on the description.
